gensim/test/test_doc2vec.py

```diff
@@ -84,10 +84,11 @@ def test_int_doctags(self):
         """Test doc2vec doctag alternatives"""
         corpus = DocsLeeCorpus()
 
-        model = doc2vec.Doc2Vec(min_count=1)
+        size = 300
+        model = doc2vec.Doc2Vec(min_count=1, size=size)
```

Comment:
There's both a 300 (count of text examples, length of `syn0`) that needs to be tested, and a 100 (dimensionality of individual vectors) - you've combined them both into `size`. These tests will pass at default (100) dimensionality, as long as you're not asserting the example-count is also 100.